.github/workflows/test-python.yaml

```diff
@@ -36,6 +36,7 @@ jobs:
 
       - name: Upload coverage to Coveralls
         uses: coverallsapp/github-action@v2
+        continue-on-error: true
```

Comment:
I've ignore failure in the coveralls action for now.
I don't think that should block PR merges, and we've also seen outages of that service before.